app/src/main/java/com/ds/avare/touch/LongTouchDestination.java

```diff
@@ -26,7 +26,8 @@
  */
 public class LongTouchDestination {
 
-    public String airport;
+    public String destinationName;
+    public String destinationType;
```

Comment:
I found having those 2 fields here is the easiest way to transit this info down the asynchronous call chain